mbuild/packing.py

```diff
@@ -172,17 +180,17 @@ def fill_box(compound, n_compounds=None, box=None, density=None, overlap=0.2,
                 for c, r in zip(compound, compound_ratio):
                     prototype_mass += r * np.sum([a.mass for a in c.to_parmed().atoms])
                 # Conversion from kg/m^3 / amu * nm^3 to dimensionless units
-                n_prototypes = int(density/prototype_mass*np.prod(box.lengths)*.60224)
+                n_prototypes = int(density/prototype_mass*np.prod(box.lengths)*0.60224)
                 n_compounds = list()
                 for c in compound_ratio:
                     n_compounds.append(int(n_prototypes * c))
 
-    # In angstroms for packmol.
+    # Convert nm to angstroms for PACKMOL.
     box_mins = box.mins * 10
     box_maxs = box.maxs * 10
     overlap *= 10
 
-    # Apply edge buffer
+    # Apply 1nm edge buffer
```

Comment:
I like spaces between numbers and units, is this common?

Reply:
[NIST recommends that as well, ill make the change.](https://physics.nist.gov/cuu/Units/checklist.html)

>There is a space between the numerical value and unit symbol, even when the value is used in an adjectival sense, except in the case of superscript units for plane angle.